src/app/iteration/iteration-modal/iteration-modal.component.html

```diff
@@ -67,3 +67,13 @@
     </div>
   </modal-footer>
 </modal>
+
+<!-- Toast notification for start iteration. -->
+<div *ngIf="isToastNotificationVisible" class="toast-pf alert alert-success alert-dismissable iteration-toast-notification">
+  <button type="button" class="close" data-dismiss="alert" aria-hidden="true" (click)="closeToastNotification()">
+    <span class="pficon pficon-close"></span>
+  </button>
+  <span class="pficon pficon-ok"></span>
+  <span class="toast-iteration-name truncate">{{iterationName}}</span> has started.
```

Comment:
Alignment is not proper

<img width="371" alt="screen shot 2017-02-07 at 11 46 56 am" src="https://cloud.githubusercontent.com/assets/10396359/22679915/58d3b98c-ed2b-11e6-98be-28c55358c060.png">